pages/docs/input-data/crac-json.md

```diff
@@ -338,23 +341,200 @@ crac.newFlowCnec()
 🔵 **iMax**:  mandatory if the FlowCnec has at least one threshold in %Imax  
 {% endcapture %}
 {% include /tabs.html id="t4" tab1name="JAVA creation API" tab1content=t4_java tab2name="JSON file" tab2content=t4_json tab3name="Object fields" tab3content=t4_objects %}
+#### AngleCnec {#angle-cnecs}
+An AngleCnec is a branch which may see a phase angle shift between its two ends when it's disconnected. This may induce insecurities in the network when it's back up. That's why we **monitor** angle CNECs and associate with them remedial actions (generally re-dispatching) that can reduce the phase angle shift between the two ends.
+
+In terms of FARAO object model, an AngleCnec is a CNEC. Even though it is associated with a branch, it is not a BranchCnec, because we cannot define on which side it is monitored: it is monitored on both sides (more specifically, we monitor the phase shift between the two sides). 
+
+An AngleCnec has the following specificities:
+
+- it contains two network elements, an importing node and an exporting node, that represent the importing and exporting ends of the branch.
+- the physical parameter which is monitored by the CNEC is the **angle**.
+- it must contain at least one threshold, defined in degrees. A threshold has a mininimum and/or a maximum value.
+- it cannot be optimized.
```

Comment:
remove this, or replace it with a note saying that CASTOR does not optimize AngleCnecs (with a link to the RAO doc). Same goes for VoltageCnecs